app/services/nomis/models/prison_offender_manager.rb

```diff
@@ -17,6 +17,7 @@ class PrisonOffenderManager
       attribute :schedule_type_description, :string
       attribute :hours_per_week, :integer
       attribute :thumbnail_id, :string
+      attribute :emails
```

Comment:
Is a type required?

Reply:
I think it behaves better without as it determines the type by what you provide it, rather than trying to force it into a string/integer etc.